pkg/ingress2httpproxy/ingress2httpproxy.go

```diff
@@ -0,0 +1,124 @@
+package ingress2httpproxy
+
+import (
+	"strings"
+
+	contour "github.com/projectcontour/contour/apis/projectcontour/v1"
+	"github.com/prometheus/common/log"
```

Comment:
std log was removed but now theres log from prometheus ;-) this should be removed - just keep logrus

Reply:
Removed prometheus